settings

```diff
@@ -1,32 +1,46 @@
-VERSION="1.23"
+VERSION="${VERSION:-1.23}"
+PROJECT="${PROJECT:-foreman}"
+RELEASEDIR="releases/$PROJECT/$VERSION"
+
+# Prepare settings file
+if [[ ! -d "$RELEASEDIR" ]] ; then
+	mkdir -p "$RELEASEDIR"
+	echo "FULLVERSION='${VERSION}.0-RC1'" > "$RELEASEDIR/settings"
+fi
+
+if [[ $PROJECT != foreman ]]; then
+  . "$RELEASEDIR/settings"
+  if [[ -z "$FOREMAN_VERSION" ]]; then
+    echo "please set FOREMAN_VERSION in $RELEASEDIR/settings"
+    exit 1
+  fi
+else
+  FOREMAN_VERSION="$VERSION"
+fi
+
 SIGNER="packages@theforeman.org"
 OSES="rhel7"
 CLIENT_OSES="rhel5 rhel6 rhel7 sles11 sles12 fedora28 fedora29"
-RELEASEDIR="releases/$VERSION"
 RPMDIR="$RELEASEDIR/rpms"
 KEYDIR="$RELEASEDIR/gnupg"
 GNUPGHOME="$KEYDIR"
 TARDIR="$RELEASEDIR/tarballs"
 EXTRASDIR="$RELEASEDIR/extras"
 HOSTS="theforeman.org,"
 DOCROOT="/var/www/vhosts/downloads/htdocs"
-PASS_NAME_GPG="theforeman/releases/foreman/$VERSION-gpg"
-PASS_NAME_KEY="theforeman/releases/foreman/$VERSION-key"
+PASS_NAME_GPG="theforeman/releases/foreman/$FOREMAN_VERSION-gpg"
+PASS_NAME_KEY="theforeman/releases/foreman/$FOREMAN_VERSION-key"
 XARGS_JOBS="-n 20 -P 4"
 GIT_DIR="$HOME/dev"
 PACKAGING_DIR="$GIT_DIR/foreman-packaging"
 PACKAGING_GIT_REMOTE=origin
 TAR_PROJECTS="foreman foreman-proxy foreman-installer foreman-selinux"
 RPM_PACKAGES="foreman foreman-installer foreman-proxy foreman-release foreman-selinux"
-ROR_BASE_TAG="foreman-rails-$VERSION"
+ROR_BASE_TAG="foreman-rails-$FOREMAN_VERSION"
```

Comment:
This and RPM_PACKAGES are a bit of a foreman specific thing. Should we also have a `releases/$PROJECT/settings` file to hold these as a layered default? Then we can set it to empty in Katello.

Reply:
done